internify/src/store/actions/studentActions.js

```diff
@@ -22,14 +27,11 @@ export const updateStudent = (data) => (dispatch) => {
   });
 };
 
-export const getStudents = () => dispatch => {
-  axios
-      .get('/api/students/')
-      .then(res => {
-          dispatch({
-              type: GET_STUDENTS,
-              payload: res.data
-          })
-      }
-      )
-};
\ No newline at end of file
+export const getStudents = () => (dispatch) => {
+  axios.get("/api/students/").then((res) => {
```

Comment:
again, can we leave this to be the original format, generally with axios calls you make each new .call on a new line.

Reply:
@tommytwm is this also from Prettier?

Reply:
Yes. Whatever Prettier makes the adjustment is what we want to follow. It just enforces consistent coding styles

Reply:
I'm going to resolve this conversation